peekingduck/pipeline/nodes/model/efficientdet_d04/efficientdet_files/detector.py

```diff
@@ -108,14 +108,15 @@ def _postprocess(
         self,
         network_output: Tuple[np.ndarray, np.ndarray, np.ndarray],
         scale: float,
-        img_shape: Tuple[int, int],
+        img_shape: Tuple[int, ...],
```

Comment:
Not sure if this change is better as it loses the contextual meaning: the original `Tuple[int, int]` says the image is 2D (height x width), but the new `Tuple[int, ...]` doesn't.
Unless there is a possibility of the image being a 3D or higher structure...?

Reply:
Made this change as I previously had a higher numpy version which typed `.shape` as `Tuple[int, ...]` instead of `Any` as the CICD version does.

Will change `img_shape` assignment to `img_shape = image.shape[0], image.shape[1]` so this can be accurately typed as `Tuple[int, int]`